src/core/app.py

```diff
@@ -1,9 +1,14 @@
 import configparser
 import tkinter as tk
+import json
 from tkinter import ttk
 
 from core import generator
+from core import tooltip
 
+# current folder is `dist`
+with open('./../constants.json', 'r') as file:
```

Comment:
```suggestion
with open("tooltips.json", "r") as file:
```

So, a few things with this: 

1. I think the file name would be more descriptive as `tooltips.json` (which I noted on that file).
2. Don't forget to use double quotes. 
3. We need to add the command to copy this file into the `dist` folder in the `build.bat` file, so that we're not referencing a file from the source code (which won't be included in the distribution). This can be done by adding the following lines to the end of the `build.bat` file:
```bat
:: Copy tooltips.json to dist
COPY ..\tooltips.json dist\tooltips.json
```
This is assuming the file name has been changed, of course. I'm not sure if the root directory should be the final home for this file, but let's assume it is for now. I can't really think of a good reason to put it anywhere else at the moment.

Reply:
A few things with this (this is going to be a wall of text, but should actually be pretty quick set of changes; sorry in advance for the long explanation):

This should not include the `./../`, because the file is in the same directory as `main.py`, which is where all references come from no matter which file you're in (assuming you're running `main.py`, which of course you would be in this case).

```suggestion
with open("tooltips_text.json", "r") as file:
```

If you have to put the relative directory navigation in, that is due to your IDE being set up to look in the wrong place, and does not reflect how it will actually run for the end user when distributed.

Also, don't forget the double quotes to be consistent with the rest of the file.

**More importantly**, I didn't realize this before, but this should really be in the `App` class's `__init__` method and the file should be loaded as an instance variable (i.e. `self.tooltips_text`). Having this outside of the class is not very Pythonic. Sorry I didn't catch that in the last review.

Finally, don't forget to add that command to the `build.bat` file that I mentioned in the previous comment. That will be needed for the program to work once built (otherwise the `tooltips_text.json` file doesn't get bundled with the software, and the whole thing breaks).